scripts/server.js

```diff
@@ -1,12 +0,0 @@
-require('dotenv').config();
-
-const path = require('path');
-const express = require('express');
-
-const assetsPath = path.join(__dirname, '../dist');
-const port = process.env.PORT || 3000;
-
-const app = express();
-app.use(express.static(assetsPath));
-
-app.listen(port, () => console.log(`Server started on port ${port}`));
```

Comment:
Why was this file deleted? 🤔

Reply:
We used to use an express server to host is. Now we host it on Cloudflare so we don't need it anymore 🎉